src/test/java/com/uwetrottmann/tmdb2/services/MoviesServiceTest.java

```diff
@@ -305,7 +305,7 @@ public void test_latest() throws IOException {
     public void test_upcoming() throws IOException {
         Call<MovieResultsPage> call = getUnauthenticatedInstance().moviesService().upcoming(
                 null,
-                null
+                "el"
```

Comment:
Any reason why you changed this from the default? If no, please revert.

Reply:
yeah got messed up after pushing a patch from master to my branch upstream, they are slightly different, so i will revert it, thanks for noticing.

Reply:
in fact it's odd, because on my branch there's no "el" :/